services/testing/nightwatch/main.go

```diff
@@ -23,56 +69,112 @@ func calcSiteURL() string {
 
 // Copy and amend .env file if it doesn't already exist
 func configureEnvFile() {
-	env := filepath.Join(conf.CoreDrupal8Path(), ".env")
+	pr := conf.GetProjectPath()
+	dr := conf.GetConfig().Drupal.Path
+	env := filepath.Join(pr, dr, "core", ".env")
 	if fs.CheckExists(env) == false {
-		generateEnvFile(env, calcSiteURL())
+		fmt.Println(Cyan(fmt.Sprintf("📋  Adding Nightwatch config to %s", env)))
+		generateEnvFile(env)
 	}
 }
 
 func yarnInstall() error {
-	d := conf.CoreDrupal8Path()
-
-	_, err := utils.CommandSubSplitOutputContext(d, "yarn", "check", "--verify-tree")
-	if err != nil {
-		fmt.Println()
-		console.Println("🧶   Nightwatch dependencies haven't been installed yet. Attempting to install with yarn", "")
-		utils.StreamOSCmdContext(d, "yarn", "install")
-	}
+	cp := conf.CoreDrupal8Path()
 
+	fmt.Println(Cyan("🦉  Ensuring Nightwatch dependencies are installed"))
+	ssh.StreamConnectCommand([]string{"cd", cp, "&&", "yarn", "install"})
 	return nil
 }
 
 func yarnTestNightwatch() {
-	d := conf.CoreDrupal8Path()
+	cp := conf.CoreDrupal8Path()
+
+	fmt.Println(Cyan("👩‍💻  Tokaido is starting a Nightwatch test run with the command `yarn test:nightwatch`"))
+	fmt.Println(Yellow(Sprintf("    If you want to run this command directly, you need run it from %s", Bold("inside the Tokaido SSH container"))))
```

Comment:
Can refer to the actual container name here?

Reply:
Yeah but I'll leave it for now - we'll probably need to replace all this messaging when we get to the final config